src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -61,7 +61,7 @@ class HttpRawlsDAO( implicit val system: ActorSystem, implicit val executionCont
         case Right(srw) =>
           logger.info("admin workspace list got: " + srw.length + " raw workspaces")
           val published = srw.collect {
-            case rw:RawlsWorkspace if rw.attributes.getOrElse(LibraryService.publishedFlag, "false").toBoolean => rw
+            case rw:RawlsWorkspace if rw.attributes.getOrElse(LibraryService.publishedFlag, AttributeBoolean(false)).asInstanceOf[AttributeBoolean].value => rw
```

Comment:
what do you want the behaviour to be here? i can do either:
a) nothing
b) throw an exception if the value isn't a bool
c) auto-convert strings "true" or "false" to bools (be liberal in what you accept), and throw an exception if the string is otherwise or the value is e.g. a number